website/api_versioned_docs/version-1.0/blockchain-integration/contracts/verifier.md

```diff
@@ -47,16 +47,31 @@ You can find detailed information in the [version management design][VersionMana
+
+- RISC Zero provides official deployments on the chains listed below.
+- Each deployment includes a [router][RiscZeroVerifierRouter.sol]. This contract implements [IRiscZeroVerifier][IRiscZeroVerifier] and can be used to verify proofs from _any_ supported version of zkVM. New versions can only be added by RISC Zero via the [timelock][TimelockController.sol] governance mechanism. Addresses for the router and timelocks are given below.
+- In addition to the router and timelock, each of the supported chains also has a [fixed-version verifier][RiscZeroGroth16Verifier.sol] contract for each supported versions of zkVM. These fixed-version verifiers are wrapped by [emergency-stop mechanisms][RiscZeroVerifierEmergencyStop.sol] (which implement [IRiscZeroVerifier][IRiscZeroVerifier]); RISC Zero can use the emergency-stop to shutdown the verifier in the event that a soundness bug is ever discovered in that particular version. Addresses for the emergency-stop contracts can be fetched by querying the router (using one of the "selector" values listed below); addresses for the underlying fixed-version verifiers can be fetched by querying the respective emergency-stop.
+
+Fixed-version contracts:
+
+| zkVM version | Contract version ("selector") |
+| ------------ | ----------------------------- |
+| 1.0 +        | `0x310fe598`                  |
+
+| Chain              | Network | RISC Zero Pubkey                                                                                         | [Timelock][TimelockController.sol]                                                                                | [Router][RiscZeroVerifierRouter.sol]                                                                              |
+| ------------------ | ------- | -------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| Ethereum           | Mainnet | [pubkey](https://etherscan.io/address/0xF616A4f81857CFEe54A4A049Ec187172574bd412)                        | [contract](https://etherscan.io/address/0x0b144E07A0826182B6b59788c34b32Bfa86Fb711#code)                          | [contract](https://etherscan.io/address/0x8EaB2D97Dfce405A1692a21b3ff3A172d593D319#code)                          |
+|                    | Sepolia | [pubkey](https://sepolia.etherscan.io/address/0x3a54a45e44a71020bd0af42063b9f23e8b9e387d)                | [contract](https://sepolia.etherscan.io/address/0xB4E3306129208cC8e6E75157f75f62eAe0B920a0#code)                  | [contract](https://sepolia.etherscan.io/address/0x925d8331ddc0a1F0d96E68CF073DFE1d92b69187#code)                  |
```

Comment:
The blank entries in the table (here and below) feels confusing to me
```suggestion
| Ethereum | Sepolia | [pubkey](https://sepolia.etherscan.io/address/0x3a54a45e44a71020bd0af42063b9f23e8b9e387d)                | [contract](https://sepolia.etherscan.io/address/0xB4E3306129208cC8e6E75157f75f62eAe0B920a0#code)                  | [contract](https://sepolia.etherscan.io/address/0x925d8331ddc0a1F0d96E68CF073DFE1d92b69187#code)                  |
```

Reply:
Meant to convey that it's a testnet, while reducing the amount of text in the table. If there's any doubt, the links point to sites that tell you what network you're looking at.

But maybe I should just add the extra labels in there?

Reply:
I wouldn't die on this hill, but IMO the it's odd to have the rows for `Ethereum Sepolia`, `Base Sepolia`, and `Arbitrum Sepolia` all look the same. It took me a while to process that the `Ethereum` label was meant to apply to both rows. 

So, I'd vote to add the extra text, but if you disagree you can just mark as resolved.